index.html

```diff
@@ -153,12 +155,14 @@ <h3>SKISIRS#2</h3>
                 </li>
             </ul>
         </section>
+
         <section class="about" id="about">
             <header class="about-us-hdr hdr">
                 <h2>ABOUT US</h2>
```

Comment:
About the text. It can be filled in via the admin panel in the future. And we don't know if the content manager will be as attentive, perhaps he will fill in everything in small letters. Therefore, in the future you can also fill in the text as if you are writing to someone, do not pay attention to the upper case, this is then done through css